### PR TITLE
React package - DateTime  installed

### PR DESCRIPTION
**Pull request by Deepti-Dk**
# Description

Installed Date-Time react package as the previous one(day picker) didn't had time option.

Fixes # (issue)

# How to test?

Through storybook

# Checklist

- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by dpfernandes — APPROVED** [review]
Approved

**Comment by daxiousm** [comment]
i reviewed it, and it looks fine

File: src/client/components/DateTimePicker/DateTimePicker.styles.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  background: #f6f6f6;
+  border: 1px solid #e8e8e8;
+  box-sizing: border-box;
+  height: 50px;
+  width: 289px;
+  border-radius: 100px;
+  text-align: center;
+}
+
+/* input {
```
Comment (Elia-Git) [inline src/client/components/DateTimePicker/DateTimePicker.styles.css]: Removing commented codes could be good practice.  Looks great !!

**Review by Elia-Git — APPROVED** [review]

**Comment by dpfernandes** [comment]
2 Approvals = It's good to Merge

File: src/client/components/DateTimePicker/DateTimePicker.styles.css (hunk 1 above)
Comment (Yaser-Aldammad) [reply src/client/components/DateTimePicker/DateTimePicker.styles.css]: That's true, removing commented codes is a good practice. I have done the same thing and was guided to keep the commented code locally, while the clean code is committed

**Review by Yaser-Aldammad — COMMENTED** [review]
That's true, removing commented codes is a good practice. I have done the same thing and was guided to keep the commented code locally, while the clean code is committed

**Pull request merged**